code/AverageCartilageThickness.m

```diff
@@ -1,5 +1,7 @@
 function averageThickness = AverageCartilageThickness(image)
     BinaryImage = image;
+    
+    BinaryImage(1:90,:)= 255;
```

Comment:
more padding added above femur for code to recognize femur correctly based on larger area of femur.